services/app-web/src/components/Header/Header.module.scss

```diff
@@ -1,54 +1,42 @@
-@import '../../styles/uswdsImports.scss';
-@import '../../styles/custom.scss';
+@use '../../styles/custom.scss' as custom;
+@use '../../styles/uswdsImports.scss' as uswds;
 
 .userInfo {
-    font-weight: font-weight('light');
-    padding: 0 units(1);
-    color: $cms-color-white;
+    font-weight: uswds.font-weight('light');
```

Comment:
I've noticed others files in our codebase use `font-weight: normal/bold` instead of the way that's used here. I'm not sure but it's probably beyond the scope of the PR to unify this?